### PR TITLE
updates some pragma versions

### PR DESCRIPTION
**Pull request by verdverm**

### CODE REVIEW

File: contracts/ens/ENSConstants.sol [hunk 1]
```diff
@@ -1,4 +1,4 @@
-pragma solidity ^0.4.18;
+pragma solidity 0.4.18;
```
Comment (sohkai) [inline contracts/ens/ENSConstants.sol]: I think this should be left unpinned; it's subclassed by `AragonApp`.

File: contracts/common/Initializable.sol [hunk 2]
```diff
@@ -1,4 +1,4 @@
-pragma solidity ^0.4.18;
+pragma solidity 0.4.18;
```
Comment (sohkai) [inline contracts/common/Initializable.sol]: This should be left unpinned; it's subclassed by `AragonApp`.

File: contracts/evmscript/EVMScriptRegistry.sol [hunk 3]
```diff
@@ -1,4 +1,4 @@
-pragma solidity 0.4.18;
+pragma solidity ^0.4.18;
```
Comment (sohkai) [inline contracts/evmscript/EVMScriptRegistry.sol]: I think this should be kept pinned; it's a pretty complicated contract.

`EVMScriptRunner` was left unpinned as it's subclassed by `AragonApp`.

**Comment by verdverm** [comment]
I noticed some of these inconsistencies, and I am still getting up to speed with Solidity dev and such. What do you think about ...?

- getting a second opinion on which one's need to be fixed to get everything straight
- getting the current contracts in the appropriate state
- documenting the guidelines for future development so things stay in line

**Comment by sohkai** [comment]
Agreed on all :).

> documenting the guidelines for future development so things stay in line

This would be awesome in a dev guide or similar! I've made https://github.com/aragon/aragonOS/issues/240 for this, feel free to add any more to the list!

**Comment by sohkai** [comment]
@izqui please re-review.

[Unpins](https://github.com/aragon/aragonOS/pull/236/commits/0ee4dd283c0ce243cee4759d8f955d2092357ff0) all the contracts in `AragonApp`'s dependency graph.

**Pull request merged**